kitsune/messages/tests/test_templates.py

```diff
@@ -23,7 +23,7 @@ def test_send_message_page(self):
 
     def _test_send_message_to(self, to):
         # Post a new message and verify it was sent.
-        data = {"to": to, "message": "hi there"}
+        data = {"to": to, "to_group": "", "message": "hi there"}
```

Comment:
Similar comment with the other test. Let's user either an empty string or None but not both